BrickController2/BrickController2/UI/Pages/ControllerProfilePage.xaml

```diff
@@ -112,6 +115,7 @@
                     </CollectionView.EmptyView>
                 </CollectionView>
 
+                <controls:FloatingActionButton Grid.Row="0" Grid.RowSpan="2" Style="{StaticResource FloatingActionButtonStyle}" Icon="playlist_add" Command="{Binding AddControllerEventWithControllerIdCommand}" ToolTipProperties.Text="{extensions:Translate AddControllerEventWithControllerId}" HorizontalOptions="Center" VerticalOptions="End" Margin="10" IsVisible="{Binding IsControllerIdSupported}"/>
```

Comment:
You can finetune button alignment if you:
- leave the same alignment: `HorizontalOptions="End"`
- apply some margin (button size is IMHO 50): e.g. `Margin="0,0,100,10"`

Just finetune it on both Windows and Android.

Reply:
Yes, looks good - thank you :)
https://github.com/vicocz/brickcontroller2/pull/75/commits/3c6cdc6d489010561b48c1c88520bee8306dc6a2